research/backend/restful/login.py

```diff
@@ -29,13 +29,11 @@ def post(self):
             return {
                 "result": 0
             }
-
         else:
             invalid_response = jsonify({
                 "result": 1,
                 "cause": "Id or password is invalid!"
              })
             invalid_response.status_code = 401
-            return invalid_response
 
-    
\ No newline at end of file
+            return invalid_response
```

Comment:
Leave one line at the end of the file next time. This rule is common for all codes here.